src/codemagic/__version__.py

```diff
@@ -1,5 +1,5 @@
 __title__ = 'codemagic-cli-tools'
 __description__ = 'CLI tools used in Codemagic builds'
-__version__ = '0.7.5'
+__version__ = '0.7.6'
```

Comment:
This PR introduces quite significant updates. I suggest bumping version up to `0.8.0`.